### DISCUSSION TITLE
Cannot install SiUSBXp.dll

### DISCUSSION THREAD

**Discussion by No-good-deed-goes-unpunished** [EvanMulawski/FanControl.CorsairLink#125] [Q&A]
Copied .dll to plugins foilder. Right click .infi, install, gives error cannot find file. I assume it's referring to this file, which does not exist.

`ServiceBinary=%10%\system32\drivers\SiUSBXp.sys`

I don't have to install iCue do I?

**Comment by EvanMulawski** [comment]
Did you extract the .zip download and install the .inf without moving it? It should copy the files in the `x64` directory to the system drivers directory.

**Comment by No-good-deed-goes-unpunished** [comment]
I extracted the contents of the archive to E:\Temp
I moved the DLL to the h:\Fan Control\Plugins folder.
I then right clicked the inf from the E:\temp folder and it comes up with an error about the .sys files missing.

I'm only trying this because fan control won't see the cooler. But my understanding is the CW-9060044-WW is not one of the hydro coolers that requires the SiUSB drivers. THe H115i RGB Pro XT was released in 2020 and IIRC Corsair dropped support for Corsairlink (where the drivers ortiginally came from), in 2018 so I would have thought coolers later than that would not need the drivers. 

Anyway, I'll try again

EDIT: Ok, I must have screwed up. It's installed now, but Fan Control still will not see the cooler.

**Reply by EvanMulawski** [reply]
Can you post the CorsairLink.log file(s)? Is this the same issue as #124?

**Comment by No-good-deed-goes-unpunished** [comment]
No, #124 seems to have been because of a faulty commander pro controller. I have removed that from the system and connected fans to the motherboard. So I closed #124. No more connect/disconnect and errors popups since removing it.

Assisted setup says no sensor responded when scanning for sensors.

I have a stack of log files with numbers eg. corsairlink.1.log etc. Can I delete those?

[CorsairLink.zip](https://github.com/EvanMulawski/FanControl.CorsairLink/files/13533523/CorsairLink.zip)

**Reply by EvanMulawski** [reply]
Delete all of them, refresh sensors in Fan Control, and attach the new one(s).

**Comment by No-good-deed-goes-unpunished** [comment]
Hi Evan, sorry for the radio silence. Had to go out of town for a few days. I decided to start from scratch. Wiped Windows, all Fan Control folders, drivers etc that were on other drives and reinstall Windows and Fan Control.

I think with all the testing and messing around that I did I must have totally bricked it.

Everything now seems to be detected and working properly, though I still see this bogus duplicate Fan #1 that spins at 4500RPM that I see on every PC that I can't control.
